locations/exporters/geojson.py

```diff
@@ -53,8 +51,10 @@ def item_to_properties(item: Item) -> dict[str, Any]:
     # Add in the extra bits
     if extras := item.get("extras"):
         for key, value in extras.items():
-            if value is not None and value != "":
-                # Only export populated values
+            if value is not None and value != "" and key != "@spider" and key != "@spider_classes":
+                # Only export populated values and ignore certain attributes
+                # which are output in the GeoJSON header as
+                # non-feature-specific attributes.
```

Comment:
This is a major change.

Reply:
The difference here is in GeoJSON output to not emit `properties["extras"]["@spider"]` for every single POI but rather to only emit `@spider` for an entire GeoJSON FeatureSet. We guarantee elsewhere in the exporter that a single GeoJSON output is not the product of two or more spiders.

Is there anyone using `properties["extras"]["@spider"]` at the moment in GeoJSON output? I can add this back in if it's potentially needed for backwards compatibility.

Reply:
Yes we do. Also everyone who uses https://www.alltheplaces.xyz/map does. Why do you need backwards compatibility when the "forward" change isn't needed nor requested?